helper/hex/hex.go

```diff
@@ -95,5 +95,9 @@ func DecodeHexToBig(hexNum string) *big.Int {
 }
 
 func DropHexPrefix(input []byte) []byte {
-	return input[2:]
+	if len(input) > 2 {
```

Comment:
Maybe we should check 
`if len(input) >= 2 && strings.ToLower(input[:2]) == "0x"` 
No need to skip first two bytes if there are no hex prefix.

Reply:
Do we need `input[:]`(this will create a new slice but on same underlying array) in else?